json_serializable/lib/src/type_helpers/config_types.dart

```diff
@@ -38,41 +39,20 @@ class KeyConfig {
 /// configuration.
 ///
 /// Values are all known, so types are non-nullable.
-class ClassConfig implements JsonSerializable {
-  @override
+class ClassConfig {
```

Comment:
This refactoring is my least favorite change.

Did you ponder just leaving `converters` empty and adding a property here?